### ISSUE TITLE
Petaluma Gap

### ISSUE THREAD

**Issue by MicheleTobias** [UCDavisLibrary/ava#335]
Michele will need to make files for this one once the documents have been published.

Start date: 1/8/2018

https://www.ttb.gov/wine/us_by_ava.shtml#California 

(b) Approved maps. The 12 United States Geological Survey (USGS) 1:24,000 scale topographic maps used to determine the boundary of the Petaluma Gap viticultural area are titled:

(1) Cotati, Calif., 1954; photorevised 1980;

(2) Glen Elle, Calif., 1954; photorevised 1980;

(3) Petaluma River, Calif., 1954; photorevised 1980;

(4) Sears Point, Calif., 1951; photorevised 1968;

(5) Petaluma Point, Calif., 1959; photorevised 1980;

(6) Novato, Calif., 1954; photorevised 1980;

(7) Petaluma, Calif., 1953; photorevised 1981;

(8) Point Reyes NE., CA, 1995;

(9) Tomales, CA, 1995;

(10) Bodega Head, Calif., 1972;

(11) Valley Ford, Calif., 1954; photorevised 1971; and

(12) Two Rock, Calif., 1954; photorevised 1971.

(c) Boundary. The Petaluma Gap viticultural area is located in Sonoma and Marin Counties in California. The boundary of the Petaluma Gap viticultural area is as described in paragraphs (c)(1) through (48) of this section:

(1) The beginning point is on the Cotati map at the intersection of Grange Road, Crane Canyon Road, and the northern boundary of section 16, T6N/R7W. From the beginning point, proceed southeast in a straight line for 1 mile, crossing over Pressley Road, to the intersection of the 900-foot elevation contour and the eastern boundary of section 16, T6N/R7W; the

(2) Proceed east-southeasterly in a straight line for 0.5 mile, crossing onto the Glen Ellen map, to the terminus of an unnamed, unimproved road known locally as Summit View Ranch Road, just north of the southern boundary of section 15, T6N/R7N; then

(3) Proceed southeast in a straight line for 0.6 mile to the intersection of Crane Creek and the 1,200-foot elevation contour, section 22, T6N/R7W; then

(4) Proceed southeast in a straight line for 2.9 miles to the marked 2,271-foot peak on Sonoma Mountain, T6N/R6W; then

(5) Proceed southeast in a straight line for 10.5 miles, crossing over the northeastern corner of the Petaluma River map and onto the Sears Point map, to the marked 682-foot summit of Wildcat Mountain; then

(6) Proceed south-southeasterly in a straight line for 3.3 miles to the intersection of State Highway 121 (also known locally as Arnold Drive) and State Highway 37 (also known locally as Sears Point Road); then

(7) Proceed east-northeasterly along State Highway 37/Sears Point Road for approximately 0.1 mile to Tolay Creek; then

(8) Proceed generally south along the meandering Tolay Creek for 3.9 miles, crossing onto the Petaluma Point map, to the mouth of the creek at San Pablo Bay; then

(9) Proceed southwesterly along the shore of San Pablo Bay for 2.7 miles, crossing the mouth of the Petaluma River, and continuing southeasterly along the bay's shoreline to Petaluma Point; then

(10) Proceed northwesterly in a straight line for 6.3 miles, crossing over the northeastern corner of the Novato map and onto the Petaluma River map, to the marked 1,558-foot peak of Burdell Mountain; then

(11) Proceed northwest in a straight line for 1.3 miles to the marked 1,193-foot peak; then

(12) Proceed west-southwesterly in a straight line for 2.2 miles, crossing onto the Petaluma map, to the marked 1,209-foot peak; then

(13) Proceed west-southwest in a straight line for 0.8 mile to the marked 1,296-foot peak; then

(14) Proceed west in a straight line for 1 mile to the marked 1,257-foot peak on Red Hill in section 31, T4N/R7W; then

(15) Proceed southwest in a straight line for 2.9 miles to the marked 1,532-foot peak on Hicks Mountain; then

(16) Proceed north-northwesterly in a straight line for 2.7 miles, crossing onto the Point Reyes NE map, to the marked 1,087-foot peak; then

(17) Proceed north-northwesterly in a straight line for 1.5 miles to the marked 1,379-foot peak; then

(18) Proceed west-northwesterly in a straight line for 2.9 miles to the marked 935-foot peak; then

(19) Proceed northwest in a straight line for 1.8 miles to the marked 804-foot peak; then

(20) Proceed west-northwesterly in a straight line for 3.1 miles, crossing onto the Tomales map, to the marked 741-foot peak; then

(21) Proceed northwesterly in a straight line for 1.3 miles to benchmark (BM) 10 on State Highway 1, at the mouth of Walker Creek in Tomales Bay; then

(22) Proceed southwesterly, then northwesterly along the shoreline of Tomales Bay to Sand Point, on Bodega Bay, and continuing northerly along the shoreline of Bodega Bay, crossing over the Valley Ford map and onto the Bodega Head map, circling the shoreline of Bodega Harbor to the Pacific Ocean and continuing northerly along the shoreline of the Pacific Ocean to the mouth of Salmon Creek, for a total of 19.5 miles; then

(23) Proceed easterly along Salmon Creek for 9.6 miles, crossing onto the Valley Ford map and passing Nolan Creek, to the second intermittent stream in the Estero Americano land grant, T6N/R10W; then

(24) Proceed east in a straight line for 1 mile to vertical angle benchmark (VABM) 724 in the Estero Americano land grant, T6N/R10W; then

(25) Proceed south-southeasterly in a straight line for 0.8 mile to BM 61 on an unmarked light duty road known locally as Freestone Valley Ford Road in the Cañada de Pogolimi land grant, T6N/R10W; then

(26) Proceed southeast in a straight line for 0.6 mile to the marked 448-foot peak in the Cañada de Pogolimi land grant, T6N/R10W; then

(27) Proceed southeast in a straight line for 0.1 mile to the northern terminus of an unnamed, unimproved road in the Cañada de Pogolimi land grant, T6N/R10W; then

(28) Proceed northeasterly, then southeasterly for 0.9 mile along the unnamed, unimproved road to the 400-foot elevation contour in the Cañada de Pogolimi land grant, T6N/R10W; then

(29) Proceed easterly along the meandering 400-foot elevation contour for 6.7 miles, crossing onto the Two Rocks map, to Burnside Road in the Cañada de Pogolimi land grant, T6N/R10W; then

(30) Proceed south on Burnside Road for 0.1 mile to an unnamed medium duty road known locally as Bloomfield Road in the Cañada de Pogolimi land grant,T6N/R9W; then

(31) Proceed southeast in a straight line for 0.6 mile to the marked 610-foot peak in the Blucher land grant, T6N/R9W; then

(32) Proceed east-southeasterly in a straight line for 0.8 mile to the marked 641-foot peak in the Blucher land grant, T6N/R9W; then

(33) Proceed northeast in a straight line for 1.2 miles, crossing through the intersection of an intermittent stream with Canfield Road, to the common Range 8⁄9 boundary; then

(34) Proceed southeast in a straight line for 0.5 mile to the marked 542-foot peak; then

(35) Proceed southeast in a straight line for 0.8 mile to the intersection of an unnamed, unimproved road (leading to four barn-like structures) known locally as Carniglia Lane and an unnamed medium duty road known locally as Roblar Road, T6N/R8W; then

(36) Proceed south in a straight line for 0.5 mile to the marked 678-foot peak, T6N/R8W; then

(37) Proceed east-southeast in a straight line for 0.8 mile to the marked 599-foot peak, T5N/R8W; then

(38) Proceed east-southeast in a straight line for 0.7 mile to the marked 604-foot peak, T5N/R8W; then

(39) Proceed east-southeast in a straight line for 0.9 mile, crossing onto the Cotati map, to the intersection of Meacham Road and an unnamed light duty road leading to a series of barn-like structures, T5N/R8W; then

(40) Proceed north-northeast along Meacham Road for 0.8 mile to Stony Point Road, T5N/R8W; then

(41) Proceed southeast along Stony Point Road for 1.1 miles to the 200-foot elevation contour, T5N/R8W; then

(42) Proceed north-northeast in a straight line for 0.5 mile to the intersection of an intermittent creek with U.S. Highway 101, T5N/R8W; then

(43) Proceed north along U.S. Highway 101 for 1.5 miles to State Highway 116 (also known locally as Graverstein Highway), T6N/R8W; then

(44) Proceed northeast in a straight line for 3.4 miles to the intersection of Crane Creek and Petaluma Hill Road, T6N/R7W; then

(45) Proceed easterly along Crane Creek for 0.8 mile to the intersection of Crane Creek and the 200-foot elevation line, T6N/R7W; then

(46) Proceed northwesterly along the 200-foot elevation contour for 1 mile to the intersection of the contour line and an intermittent stream just south of Crane Canyon Road, T6N/R7W; then

(47) Proceed east then northeasterly along the northern branch of the intermittent stream for 0.3 mile to the intersection of the stream with Crane Canyon Road, T6N/R7W; then

(48) Proceed northeasterly along Crane Canyon Road for 1.2 miles, returning to the beginning point.

[T.D. TTB-149, 82 FR 57662, Dec. 7, 2017]

**Comment by MicheleTobias** [comment]
I added this one on 2018-02-12 so that the dataset would be complete for the fire boundaries from this past summer. While Petaluma Gap didn't technically exist then as an AVA, there are presumably wineries there that would have been affected.

**Comment by SoominHur** [comment]
Reviewed.
- Edit the typo on approved map (2)'s map name; Glen Elle -> Glen Ellen
- Edit the line proceeded along the private road beside Tolay Creek to (8) "Proceed ... along the meandering Tolay Creek" 
- Extend the line along the unnamed, unimproved road little more, so it can proceed not only northeasterly, but also (28) "southeasterly." 
- Edit the line passing the intersection with an unnamed intermittent stream and unimproved/unnamed road to pass the (33) "intersection of an intermittent stream with Canfield Road." Thus, the distance of line between the intersection point with Ranges 8/9 and (34) "marked 542-foot peak" can be approximately 0.5 mile in a straight line.

**Comment by MicheleTobias** [comment]
@sdelong reported the following points to check in Issue #386 :
 * (21) The wrong BM 10 is plotted - there is another BM 10 at the mouth of the Walker River, which is the correct one.
 * (5) The wrong section 34 southwest corner has been plotted - the correct one is approx 5 miles west T24S, R9E.

I'm reopening the issue so we can check these out and make sure the boundary is as correct as we can make it.

**Comment by MicheleTobias** [comment]
@sdelong I found the error in step 21 and corrected it (there were two BM 10s along the highway in close proximity).

I was unable to figure out what was wrong with step 5.  It says "(5) Proceed southeast in a straight line for 10.5 miles, crossing over the northeastern corner of the Petaluma River map and onto the Sears Point map, to the marked 682-foot summit of Wildcat Mountain; then"  I searched the whole description for reference to section 34 and didn't find any, so I think I need some guidance on what's not right.  Maybe it was a different AVA?
